### PR TITLE
Add the payload diff feature

### PR DESCRIPTION
**Pull request by mallow111**
Replace checkbox with radio button to select payload on left and right side.
Screen Shot 2019-11-16 at 1 00 16 PM

### CODE REVIEW

File: pkg/sloop/webfiles/resource.html [hunk 1]
```diff
@@ -12,14 +12,19 @@
     <link rel="stylesheet" type="text/css" href="/webfiles/sloop.css">
     <link rel="stylesheet" type="text/css" href="/webfiles/resource.css">
     <link href="https://unpkg.com/nprogress@0.2.0/nprogress.css" rel="stylesheet" />
+    <link rel="stylesheet" type="text/css" href="http://cemerick.github.io/jsdifflib/diffview.css"/>
```
Comment (nurland) [inline pkg/sloop/webfiles/resource.html]: nit: http->https

File: pkg/sloop/webfiles/resource.html [hunk 2]
```diff
@@ -12,14 +12,19 @@
...
+
     <title>Resource {{.Kind}}/{{.Namespace}}/{{.Name}}</title>
 
     <script src="https://unpkg.com/axios/dist/axios.min.js" type="text/javascript"></script>
     <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js" type="text/javascript"></script>
     <script src="https://unpkg.com/nprogress@0.2.0/nprogress.js" type="text/javascript"></script>
+    <script type="text/javascript" src="http://cemerick.github.io/jsdifflib/diffview.js"></script>
+    <script type="text/javascript" src="http://cemerick.github.io/jsdifflib/difflib.js"></script>
```
Comment (nurland) [inline pkg/sloop/webfiles/resource.html]: nit: http->https
please also move type after src just for consistency with includes above

File: pkg/sloop/webfiles/resource.html [hunk 3]
```diff
@@ -38,28 +43,54 @@ <h2>Links</h2>
...
-                <td v-for="item in selectedPayloads.slice(0,2)" style="vertical-align: top;">
-                    <pre>${ item | pretty }</pre>
-                </td>
-            </tr>
-        </table>
     </div>
 
 </div>
+
+<script type="text/javascript">
```
Comment (nurland) [inline pkg/sloop/webfiles/resource.html]: please merge with the `<script>` section below.

File: pkg/sloop/webfiles/resource.html [hunk 4]
```diff
@@ -122,10 +152,12 @@ <h2>Viewing range {{.ClickTime}} +/- {{.PlusMinusTime}}</h2>
...
             this.currentSortBy = sortBy;
         },
-        check: function(e) {
-            if (e.target.checked) {
-                console.log(e.target.id)
-                console.log(e.target.value)
+        pretty: function(value) {
+            return JSON.stringify(JSON.parse(value), null, 2);
+        },
+        check: function() {
```
Comment (nurland) [inline pkg/sloop/webfiles/resource.html]: nit: `check`->`diff` (or `doDiff`).

**Comment by mallow111** [comment]
@nurland  have addressed your comments, thank you for your input!

File: pkg/sloop/webfiles/resource.html [hunk 5]
```diff
@@ -11,15 +11,20 @@
     <link rel='shortcut icon' type='image/x-icon' href='/webfiles/favicon.ico'/>
     <link rel="stylesheet" type="text/css" href="/webfiles/sloop.css">
     <link rel="stylesheet" type="text/css" href="/webfiles/resource.css">
-    <link href="https://unpkg.com/nprogress@0.2.0/nprogress.css" rel="stylesheet" />
+    <link rel="stylesheet" href="https://unpkg.com/nprogress@0.2.0/nprogress.css"  />
+    <link rel="stylesheet" type="text/css" href="https://cemerick.github.io/jsdifflib/diffview.css"  />
```
Comment (nurland) [inline pkg/sloop/webfiles/resource.html]: nit: trailing spaces.

**Pull request merged**